docs/src/modules/java/pages/author-your-first-service.adoc

```diff
@@ -55,17 +54,24 @@ mvn archetype:generate ^
 ----
 --
 
-. Fill in
-* groupId: `com.example`
-* artifactId: `helloworld`
-* version: `1.0-SNAPSHOT`
-* package: `helloword`
++
```

Comment:
Putting the correct continuation `+`s is fiddly here. I added some back that you removed.